packages/webapp/webapp_server.js

```diff
@@ -818,9 +816,16 @@ WebAppInternals.setInlineScriptsAllowed = function (value) {
   WebAppInternals.generateBoilerplate();
 };
 
+// this is the default implementation, can be overridden below
+WebAppInternals.bundledJsCssUrlRewriteHook = function (url) {
```

Comment:
Why expose this on the `WebAppInternals` namespace instead of keeping it private to the scope of this module?